src/command_dynamic.cc

```diff
@@ -345,8 +355,10 @@ system_method_redirect(const torrent::Object::list_type& args) {
   std::string new_key  = torrent::object_create_string(args.front());
   std::string dest_key = torrent::object_create_string(args.back());
 
-  rpc::commands.create_redirect(create_new_key(new_key), create_new_key(dest_key),
-                                rpc::CommandMap::flag_public_xmlrpc | rpc::CommandMap::flag_delete_key | rpc::CommandMap::flag_modifiable);
+  rpc::commands.create_redirect(create_new_key(new_key),
+                                create_new_key(dest_key),
+                                rpc::CommandMap::flag_public_xmlrpc | rpc::CommandMap::flag_delete_key |
+                                  rpc::CommandMap::flag_modifiable);
```

Comment:
70 char lines is a relic of ancient coding practices, it doesn't make sense for such verbose and namespace heavy cases as this.

Breaking up lines like this just makes it worse.

Should either use newline for all, keep it on one line or respect the choice made by the programmer.